<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>290</width>
    <height>968</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Style Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_1">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_1">
     <item>
      <widget class="QComboBox" name="comboPresets">
       <property name="toolTip">
        <string>Fill the values below from a stored style preset</string>
       </property>
       <item>
        <property name="text">
         <string>Load preset</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Saves the current style as a preset</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="gtk-save">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tabs">
     <widget class="QWidget" name="tab_Shape">
      <attribute name="title">
       <string>Shape</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox_1">
         <property name="title">
          <string>Shape Appearance</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_1" columnstretch="1,0" columnminimumwidth="0,100">
          <item row="0" column="0">
           <widget class="QLabel" name="label_ShapeColor">
            <property name="text">
             <string>Shape color</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="Gui::ColorButton" name="ShapeColor">
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_AmbientColor">
            <property name="text">
             <string>Ambient shape color</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="Gui::ColorButton" name="AmbientColor">
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_EmissiveColor">
            <property name="text">
             <string>Emissive shape color</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="Gui::ColorButton" name="EmissiveColor">
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_SpecularColor">
            <property name="text">
             <string>Specular shape color</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="Gui::ColorButton" name="SpecularColor">
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_Transparency">
            <property name="text">
             <string>Shape transparency</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QSpinBox" name="Transparency">
            <property name="suffix">
             <string notr="true"> %</string>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_Shininess">
            <property name="text">
             <string>Shape shininess</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QSpinBox" name="Shininess">
            <property name="suffix">
             <string notr="true"> %</string>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Other</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2" columnstretch="1,0" columnminimumwidth="0,100">
          <item row="0" column="0">
           <widget class="QLabel" name="label_LineColor">
            <property name="text">
             <string>Line color</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="Gui::ColorButton" name="LineColor">
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_LineWidth">
            <property name="text">
             <string>Line width</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="LineWidth">
            <property name="suffix">
             <string> px</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_PointColor">
            <property name="text">
             <string>Point color</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="Gui::ColorButton" name="PointColor">
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_PointSize">
            <property name="text">
             <string>Point size</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QSpinBox" name="PointSize">
            <property name="suffix">
             <string> px</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_DrawStyle">
            <property name="text">
             <string>Draw style</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QComboBox" name="DrawStyle">
            <item>
             <property name="text">
              <string>Solid</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Dashed</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Dotted</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>DashDot</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_DisplayMode">
            <property name="text">
             <string>Display mode</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QComboBox" name="DisplayMode">
            <item>
             <property name="text">
              <string>Flat Lines</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Shaded</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Wireframe</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Points</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_1">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_Annotation">
      <attribute name="title">
       <string>Annotation</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_Signpost">
       <item>
        <widget class="QLabel" name="label_Signpost">
         <property name="text">
          <string>Annotation settings are now managed by the Annotation Style Manager.</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_OpenManager">
         <property name="text">
          <string>Open Annotation Style Manager</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="applyButton">
       <property name="toolTip">
        <string>Apply the above style to selected object(s)</string>
       </property>
       <property name="text">
        <string>Selected</string>
       </property>
       <property name="icon">
        <iconset theme="gtk-apply">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Gui::InputField</class>
   <extends>QLineEdit</extends>
   <header>Gui/InputField.h</header>
  </customwidget>
  <customwidget>
   <class>Gui::ColorButton</class>
   <extends>QPushButton</extends>
   <header>Gui/Widgets.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
